### PR TITLE
Add SteeringWheel.Heating

### PR DESCRIPTION
**Pull request by erikbosch**
Inspired by HVAC_STEERING_WHEEL_HEAT in https://android.googlesource.com/platform/hardware/interfaces/+/master/automotive/vehicle/2.0/types.hal

### CODE REVIEW

**Comment by erikbosch** [comment]
Meeting notes:

- Please review
- Nick: Maybe call it something with both heat and cooling.
- Pierre: Check if it can be aligned with seat heating/cooling (reusable signal)

**Comment by erikbosch** [comment]
AP: Erik to update PR

**Comment by SebastianSchildt** [comment]
Meeting 08/15 

 - Is "TemperatureControl" a good name (it sounds like HVAC, but this signal is not very usable for that (relative scale))
 - Is "Control" is a good suffix
 - Maybe something something with "Relative" e.g. RelativeTemperature

File: spec/include/ItemHeatingCooling.vspec [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+
+
+HeatingCooling:
```
Comment (erikbosch) [inline spec/include/ItemHeatingCooling.vspec:7]: Changed `TemperatureControl` (in previous PR version) to `HeatingCooling`, I think that I prefer that over `RelativeTemperature` but we can discuss that at next meeting

**Comment by erikbosch** [comment]
Meeting notes:

- Erik changed name to "HeatingCooling"
- Daniel: Would not use term "temperature" if unit is not celsius, possibly better with "intensity"
- Sebastian: Possibly "HeatingIntensity"
- Erik: Please review/discuss until next week

**Comment by erikbosch** [comment]
Meeting notes: merge

**Pull request merged**